### PR TITLE
Adding SetBit to bitmanipulation

### PR DESCRIPTION
**Pull request by lukasb1b**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
Please fix PR checks

**Comment by lukasb1b** [comment]
Im sorry im new to github and I accedently resubmittet it. Can you explain to me how I need to change it?

**Comment by siriak** [comment]
You can find problems here https://github.com/TheAlgorithms/Java/actions/runs/6060803308/job/16453536079?pr=4348
These are indentation and spaces
image

**Comment by lukasb1b** [comment]
I now changed the formatting i hope thats it

**Comment by siriak** [comment]
Not yet :) See output of the failing job

File: src/test/java/com/thealgorithms/bitmanipulation/SetBit.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.thealgorithms.bitmanipulation;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+
```
Comment (siriak) [inline src/test/java/com/thealgorithms/bitmanipulation/SetBit.java:7]:
```suggestion
```

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**